pkg/agent/handler/accessrequest.go

```diff
@@ -58,7 +58,11 @@ func (h *accessRequestHandler) Handle(action proto.Event_Type, meta *proto.Event
 	if ok := shouldProcessPending(ar.Status.Level, ar.Metadata.State); ok {
 		log.Tracef("access request handler - processing resource in pending status")
 		ar := h.onPending(ar)
-		return h.client.CreateSubResourceScoped(ar.ResourceMeta, ar.SubResources)
+		err := h.client.CreateSubResourceScoped(ar.ResourceMeta, ar.SubResources)
+		if err != nil {
+			return err
+		}
+		return h.client.CreateSubResourceScoped(ar.ResourceMeta, map[string]interface{}{"status": ar.Status})
```

Comment:
How come we need it for access request and managed app? The other sub resource is x-agent-details

Reply:
For consistency